### DISCUSSION TITLE
Confusing version "latest" in doc pages

### DISCUSSION THREAD

**Discussion by FilipSLO** [micropython/micropython#12634] [General]
[Documentation page](https://docs.micropython.org/en/latest/) has version "latest" which is confusing for me.
I already reported a false bug as I thought that it means **latest release** not **development** version.

Can you please reword version "**latest**" it a bit clearer?

**Comment by dpgeorge** [comment]
Which part exactly is confusing?

At the very top of the main page for the documentation it says:
```
This is the documentation for the latest development branch of MicroPython and may refer to features that are not available in released versions.
```
That's quite clear that it's the "latest development branch".

**Reply by FilipSLO** [reply]
True for the top right side of the doc.
I am really sorry for my confusion but I somehow overlooked that and saw only bottom left corner: 
![image](https://github.com/micropython/micropython/assets/73668695/1279bbf0-bddb-4743-9330-19626f49d8d0)

**Reply by dpgeorge** [reply]
That part in the bottom corner needs to be short because there's not much space for it.  I can see how it may be confused as being the "latest version", rather than the "latest development release".

**Reply by FilipSLO** [reply]
Maybe I should try harder and give concrete suggestions:
- unreleased
- development 
- unfinished
- unstable
- WIP
- master
- head
- next
- future
- undefined
- not yet
- None (pun intended ;) as (u)python is fun not just useful)

HTH and thanks to all contributors especially you @dpgeorge!
